README.md

```diff
@@ -88,116 +59,108 @@ Key features:
 * Enchanting support
 * Support for adding and customizing materials, patterns and gems entirely trough config files
 
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-### Built With
-
-This project is created with Fabric, and its toolchain
-
-* [Fabric mod template](https://github.com/FabricMC/fabric-example-mod)
-* [Fabric API](https://github.com/FabricMC/fabric)
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
 
+<!-- ROADMAP -->
 
-<!-- GETTING STARTED -->
+## Roadmap
 
-## Getting Started with modding
+Forgero is in active development, and features are being added frequently.  
+We support [Semantic Versioning](https://semver.org/), meaning we make sure we break your saves as little as possible
+with updates, and if possible only in major releases.
```

Comment:
We support [Semantic Versioning](https://semver.org/), which means we make sure to not introduce breaking changes unless we are moving to a new major version.

Reply:
That is a much better way to word it, thanks!